snetd/cmd/components.go

```diff
@@ -141,6 +142,14 @@ func (components *Components) EtcdClient() *etcddb.EtcdClient {
 	return components.etcdClient
 }
 
+func (components *Components) PrefixedAtomicStorage() *escrow.PrefixedAtomicStorage {
```

Comment:
Let's rename it to something less abstract for instance `EtcdLockerStorage` or `LockerStorage`. `PrefixedAtomicStorage` type is used few times in code for different storages.

Reply:
LockerStorage would be ideal i think. And changed the code accordingly.